packages/guest-submissions-block/src/Components/Headline/ModalHeadline.tsx

```diff
@@ -22,7 +22,7 @@ export const ModalHeadline: FC<BlockProps> = ({ appBridge }) => {
             content={modalcontent}
             onTextChange={onTextChange}
             defaultValue={''}
-            placeholder={'note / description'}
+            placeholder={'Instructional text'}
```

Comment:
```suggestion
            placeholder="Instructional text"
```

Reply:
Correct: we can treat this prop as a plain string. But, looking towards i18n, we’d probably end up needing the curly braces again :D

Reply:
I’ll change it to be correct *now*.